projects/packages/forms/src/abilities/class-forms-abilities.php

```diff
@@ -0,0 +1,455 @@
+			array(
+				'label'               => __( 'Delete Form Submission', 'jetpack-forms' ),
+				'description'         => __( 'Permanently delete a form submission. This action cannot be undone. Use update-submission with status "trash" for reversible deletion.', 'jetpack-forms' ),
+				'category'            => self::CATEGORY_SLUG,
+				'input_schema'        => array(
+					'type'                 => 'object',
+					'required'             => array( 'id' ),
+					'properties'           => array(
+						'id' => array(
+							'type'        => 'integer',
+							'description' => __( 'The submission ID to permanently delete.', 'jetpack-forms' ),
+						),
+					),
+					'additionalProperties' => false,
+				),
+				'execute_callback'    => array( __CLASS__, 'delete_form_submission' ),
+				'permission_callback' => array( __CLASS__, 'can_delete_posts' ),
+				'meta'                => array(
+					'annotations'  => array(
+						'readonly'    => false,
+						'destructive' => true,
+						'idempotent'  => false,
+					),
+					'show_in_rest' => true,
+				),
+			)
+		);
+	}
+
+	/**
+	 * Register ability to get status counts.
+	 *
+	 * @return void
+	 */
+	private static function register_get_status_counts_ability() {
+		wp_register_ability(
+			'jetpack-forms/get-status-counts',
+			array(
+				'label'               => __( 'Get Submission Status Counts', 'jetpack-forms' ),
+				'description'         => __( 'Get a summary of form submissions grouped by status. Returns counts for inbox (active), spam, and trash. Useful for dashboard stats or checking if there are new submissions.', 'jetpack-forms' ),
+				'category'            => self::CATEGORY_SLUG,
+				'input_schema'        => array(
+					'type'                 => 'object',
+					'default'              => array(),
+					'properties'           => array(
+						'search'    => array(
+							'type'        => 'string',
+							'description' => __( 'Only count submissions matching this search term.', 'jetpack-forms' ),
+						),
+						'parent'    => array(
+							'type'        => 'integer',
+							'description' => __( 'Only count submissions from a specific page or post.', 'jetpack-forms' ),
+						),
+						'before'    => array(
+							'type'        => 'string',
+							'description' => __( 'Only count submissions before this date (ISO8601 format).', 'jetpack-forms' ),
+							'format'      => 'date-time',
+						),
+						'after'     => array(
+							'type'        => 'string',
+							'description' => __( 'Only count submissions after this date (ISO8601 format).', 'jetpack-forms' ),
+							'format'      => 'date-time',
+						),
+						'is_unread' => array(
+							'type'        => 'boolean',
+							'description' => __( 'Set true to count only unread, false for only read.', 'jetpack-forms' ),
+						),
+					),
+					'additionalProperties' => false,
+				),
+				'execute_callback'    => array( __CLASS__, 'get_status_counts' ),
+				'permission_callback' => array( __CLASS__, 'can_edit_pages' ),
```

Comment:
The fact that we are creating functions just to do current_user_can, suggests that we should probably expand the API and not only have a callback but also a way to specify the capabilities needed to avoid this boilerplate. What do you think @JasonTheAdams, @emdashcodes ?